lab_2_keywords_cooccurrence/main.py

```diff
@@ -40,7 +64,16 @@ def calculate_frequencies_for_content_words(candidate_keyword_phrases: KeyPhrase
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not check_list(candidate_keyword_phrases, tuple, False):
+        return None
+    tokens = []
+    for phrase in candidate_keyword_phrases:
+        for word in phrase:
+            tokens.append(word)
+    frequencies = {}
```

Comment:
this dict could be created in one line and return without the `frequencies` variable